src/backend/integrations/builtin/obs/effects/toggle-obs-source-visibility.ts

```diff
@@ -31,7 +32,30 @@ export const ToggleSourceVisibilityEffectType: Firebot.EffectType<EffectProperti
         categories: ["common"]
     },
     optionsTemplate: `
-<eos-container header="Sources">
+<eos-container ng-show="orphanedSources.length > 0">
+        <div class="effect-info alert alert-warning">
+            <p><b>Warning!</b> 
+                Cannot find {{orphanedSources.length}} sources in this effect. Ensure the correct profile or scene collection is loaded in OBS, and OBS is running.
+            </p>
+        </div>
+</eos-container>
+<setting-container ng-show="orphanedSources.length > 0" header="Missing Sources ({{orphanedSources.length}})" collapsed="true">
+    <div ng-repeat="sceneName in orphanedSources track by $index">
+      <div class="list-item" style="display: flex;border: 2px solid #3e4045;box-shadow: none;border-radius: 8px;padding: 5px 5px;">
+        <div class="pl-5">
+          <span>Scene: {{sceneName.sceneName}},</span>
+            <span ng-if="sceneName.sourceName != null">Name: {{sceneName.sourceName}},</span>
+            <span>Id: {{sceneName.sourceId}},</span>
```

Comment:
I don't think we need an ID here. Most users don't utilize this kind of data unless they come to us or are looking to solve the problem themselves, in which case the logical place would be to look at the logs.

Reply:
this is for backwards compatibility we currently dont store the name only the Scene and ID

Reply:
![{F43554AD-5F1D-4A58-A6CE-D671715FDCD0}](https://github.com/user-attachments/assets/5ac0f8e3-fb1c-4a63-b398-ebcaeb2354b4)
if i do remove the ID this is what we will end up with

Reply:
![{EE694A15-6C34-4E10-93AF-1DE6017EE583}](https://github.com/user-attachments/assets/1d99e873-e905-4d16-93e0-3a397c0cd9f5)
if you ok with this i think this is the best of both worlds new data will only show the name and old data will show the id and Unknown name

Reply:
Yes this looks great!